src/Service/station/ServiceStation.java

```diff
@@ -0,0 +1,34 @@
+package Service.station;
+
+import Service.transport.general.Transport;
+
+public class ServiceStation {
+    public void check(Transport transport) {
+        System.out.println("Обслуживаем " + transport.getModelName());
+        for (int i = 0; i < transport.getWheelsCount(); i++) {
+            transport.updateTyre();
+            System.out.print(" № " + (i + 1) + " у " + transport.getModelName());
+            System.out.println();
+        }
+        transport.checkEngine();
+        transport.checkTrailer();
+    }
+    public void check(Transport transport1, Transport transport2) {
```

Comment:
А для чего метод с двумя транспортами? Ну даже если и нужен, то от дублирования кода нужно избавляться
```suggestion
    public void check(Transport transport1, Transport transport2) {
         check(transport1);
         check(transport2);
    }
```